app/src/containers/Workspaces/AcceptInvitation.tsx

```
@@ -0,0 +1,213 @@
+    <div className="h-screen overflow-hidden flex w-full min-h-screen items-center justify-center relative">
+      <div className="absolute w-full h-full bg-primary z-[-2]" />
+      <div className="w-[762px] mx-auto my-0 text-center bg-white p-10 relative rounded-[5px]">
+        <div>
+          <Logo />
+        </div>
+        <div className="font-bold text-[26px] leading-9 text-sapphire-blue mt-[3px]">
+          {t('Accept_invitation.title')}
+        </div>
+        <p className="text-[14px] leading-6 text-sapphire-blue max-w-[567px] mx-auto mt-6 mb-10">
+          <Trans
+            components={[<strong></strong>]}
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)